tests/MiddlewareRunnerTest.php

```diff
@@ -90,6 +90,42 @@ public function testProcessStack(array $middlewares, $expectedCallCount)
         }
     }
 
+    public function testCorruptMiddlewareStack()
```

Comment:
The method name seems to imply something is "corrupt" here, can you update this to emphasize what is being tested here? :-) Otherwise LGTM :+1:

Reply:
@clue fixed :+1: